File: antigravity/a-vanilla/run-3/src/orm/member.py
from tina4_python.orm import ORM, IntegerField, StringField, has_many

class Member(ORM):
    table_name = "members"

    id = IntegerField(primary_key=True, auto_increment=True)
    name = StringField(required=True)
    email = StringField(required=True)
    join_date = StringField(required=True)

    # Relationship
    loans = has_many("Loan", foreign_key="member_id")

File: antigravity/archive/run-3-php-v2.0/src/orm/Member.php
<?php

use Tina4\ORM;

class Member extends ORM
{
    public string $tableName = "members";
    public string $primaryKey = "id";
    public bool $autoMap = true;

    public int $id;
    public string $name;
    public string $email;
    public string $joinDate;
    public string $createdAt;
    public string $updatedAt;

    /**
     * Get borrowing history for this member
     * @return Loan[]
     */
    public function loans(): array
    {
        return $this->hasMany(Loan::class, "member_id");
    }
}
